checker/src/main/java/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -293,6 +295,13 @@ public Void visitMethod(MethodTree tree, Void p) {
         return super.visitMethod(tree, p);
     }
 
+    @Override
+    public Void visitThrow(ThrowTree tree, Void p) {
```

Comment:
Why override the method? The implementation looks exactly the same.

Reply:
Already removed it.